source/load-balancers/load-balancers.rst

```diff
@@ -0,0 +1,531 @@
+
+All services behind a load balancer that have been started with the aforementioned option MUST
+add a top level :code:`serviceId` field to their response to the :code:`hello` command. This field
+MUST be a BSON :code:`ObjectId` and SHOULD NOT change while the service is running.
+When a driver is configured to not be in load balanced mode and the service is configured behind
+a load balancer, the service MAY return an error from the driver's :code:`hello` command that
+the driver is not configured to use it properly.
+
+All services that have the behaviour of reaping idle cursors after a specified period of time MAY
+also close the connection associated with the cursor when the cursor is reaped. Conversely, those
+services MAY reap a cursor when the connection associated with the cursor is closed.
+
+All services that have the behaviour of reaping idle transactions after a specified period of time
+MAY also close the connection associated with the transaction when the transaction is reaped.
+Conversely, those services must abort a transaction when the connection associated with the
+transaction is closed.
+
+Any applications that connect directly to services and not through the load balancer MUST also
+supply the :code:`loadBalanced=true` option to the driver they use to connect.
+
+
+Q&A
+---
+
+Why use a connection string option instead of a new URI scheme?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use of a connection string option would allow the driver to continue to use SRV records that
+pointed at a load balancer instead of a replica set without needing to change the URI provided
+to the :code:`MongoClient`. The SRV records could also provide the default :code:`loadBalanced=true`
+in the TXT records.
+
+Why explicitly opt-in to this behaviour instead of letting mongos inform the driver of the load balancer?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This was a case of complexity in implementation and the number of affected services, including
+but not limited to mongos, Atlas Proxy, Elastic Front-End, and Data Lake. All services would need
+to be modified to understand the PROXY protocol (although Data Lake already does), or add
+configuration options that are read during start-up. From there they would need to alter their
+hello responses in order to tell the driver they were behind a load balancer.
+
+From a driver's view there is a change in behaviour in that once in load balanced mode, the driver
+can never transition out of that mode, unlike other states in the SDAM specification.
+
+Why does this specification instruct drivers to not check connections back into the connection pool in some circumstances?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case of a load balancer fronting multiple services, it is possible that a connection to th
```

Comment:
Typo: there's a space between "th" and "e".

Reply:
Fixed.